cypress/integration/show_work_orders.spec.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+describe('Show all work orders', () => {
+  it('Displays work order details, reference, date raised, last update, priority, property and description', () => {
+    //Stub request with work orders response
+    cy.login()
+    cy.server()
+    cy.fixture('work_orders/jobs.json').as('workorderslist')
+    cy.route('GET', 'api/repairs', '@workorderslist')
+    cy.visit(`${Cypress.env('HOST')}/repairs/jobs`)
+
+    cy.get('.govuk-table').within(() => {
+      cy.contains('th', 'Reference')
+      cy.contains('th', 'Date raised')
+      cy.contains('th', 'Last update')
+      cy.contains('th', 'Priority')
+      cy.contains('th', 'Property')
+      cy.contains('th', 'Description')
+    })
+
+    //can see reference number/ date raised, last updated, priority, property address, description
+    cy.get('.govuk-table__cell').within(() => {
```

Comment:
Now that i'm looking at the HTML generated, shouldn't this be under `cy.get('.govuk-table__row')` ?
There are a bunch of `govuk-table__cell` classes so i'm not sure this would be the right scope to look within.
Might even be worth having the word order reference id as a data attribute so it's easier to select the relevant row with this unique identifier